static/index.css

```
@@ -32,17 +32,38 @@ body {
   width: 100vw;
   height: 100vh;
 
-  background-color: var(--color-error-default);
+  background-color: #d03242;
   font-family: Euclid, Roboto, Helvetica, Arial, sans-serif;
 }
 
-p {
-  margin: 2em;
-}
-
 p a {
   text-decoration: underline;
-  color: var(--color-primary-default);
+  color: white;
```

Can we use design token CSS vars here?

```suggestion
  color: var(--color-error-inverse);
```

---

Yes, seen that little late, will fix it